Core/GDCore/Project/MeasurementUnit.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ * GDevelop Core
+ * Copyright 2008-present Florian Rival (Florian.Rival@gmail.com). All rights
+ * reserved. This project is released under the MIT License.
+ */
+#include "MeasurementUnit.h"
+#include "GDCore/Serialization/SerializerElement.h"
+#include "GDCore/String.h"
+#include <vector>
+
+namespace gd {
+
+MeasurementUnit::~MeasurementUnit() {}
+
+gd::MeasurementUnit MeasurementUnit::unknown = CreateUnknown();
```

Comment:
Properties are declared using something like a builder. The unit may not be explicitly declared. When this happens we shouldn't make assumption on the unit. For instance, in the future, when we do a dimensional analyzer, an unknown unit will allow to know that we must not warn the user about dimensional issues because we don't know.

Reply:
ok 👍 
Or should it be named, `Unset` ? rather than unknown?

Reply:
I renamed it "undefined".

Reply:
nice!